### PR TITLE
Fixing gnomAD plots JSON structure

### PR DESCRIPTION
**Pull request by avvaruakshay**

### CODE REVIEW

**Comment by vincerubinetti** [comment]
There aren't really any frontend/site changes here, so my review isn't as useful.

However as a side note, I've noticed that these PRs have had lots of files without the EOF newline:
Screenshot 2024-12-17 at 2 42 17 PM

This isn't a big deal practically speaking, but I'd recommend everyone on the team get in the habit of adding this, and moreover, formatting your code with an opinionated formatter like Black for Python.

**Review by hdashnow — CHANGES_REQUESTED** [review]
This looks good on the preview. 
Could you please add the logic/list of reliable genes somewhere so we can have a record and update it as needed?

**Comment by avvaruakshay** [comment]
I have added the reliability values to the code as a dictionary, for now. It can be updated as when the reliability status changes.

File: scripts/gnomad-json.py [hunk 1]
```diff
@@ -5,6 +5,66 @@
 from statsmodels.stats import proportion
 import argparse
 
+reliability = {
```
Comment (hdashnow) [inline scripts/gnomad-json.py:8]: These loci are in the TSV data file but not in the reliability dictionary:
DMD
RILPL1
XYLT1

**Review by hdashnow — CHANGES_REQUESTED** [review]
There are 3 loci missing from the reliability dictionary. What did you all decide about these?

**Review by hdashnow — APPROVED** [review]

**Pull request merged**